web/cypress/support/page_objects/profilePage.ts

```diff
@@ -89,6 +89,14 @@ export class ProfilePage extends OnboardingPage {
   getHomeBased(radio?: boolean) {
     return cy.get(`input[name="home-based-business"]${radio === undefined ? "" : `[value="${radio}"]`}`);
   }
+
+  getTaxFilingCalendar() {
+    return cy.get('[data-testid="filings-calendar"]');
+  }
+
+  registerForTaxes() {
+    return cy.get('[data-testid="register-for-taxes"]').first().click();
+  }
```

Comment:
[sand] A slightly better place for these functions might be the `dashboardPage.ts` file in the same directory.